lionblog-seongil/src/routes/PostEditPage.jsx

```diff
@@ -1,5 +1,63 @@
+import posts from "../data/posts";
+import { useParams } from "react-router-dom";
 const PostEditPage = () => {
-  return <div>Post Edit Page</div>;
+  const handleSignUpSubmit = () => {
+    alert("수정 완료");
+  };
+  const postId = useParams().postId;
+  const post = posts.find((post) => post.id === parseInt(postId));
+  return (
+    <div className="flex flex-col items-center w-1/2">
+      <h3 className="font-bold text-2xl">게시글 수정</h3>
+      <form className="form gap-2" onSubmit={handleSignUpSubmit}>
+        <label required type="text" className="label">
+          제목:
+        </label>
+        <input
+          required
+          value={post.title}
```

Comment:
이렇게 설정하면 아마 값 수정이 안 될 거예요! defaultValue 속성을 사용해 봐요 ~
```suggestion
          defaultValue={post.title}
```